packages/api/src/command/medical/document/document-redownload.ts

```diff
@@ -140,7 +158,7 @@ async function processDocsOfPatient({
 
   try {
     log(`Processing ${docs.length} documents for patient ${patientId}...`);
-    await updateDocQuery({
+    await appendDocQueryProgress({
```

Comment:
why is this changing? given the amount of pain we had in this part of the code, it seems in our best interest to leave as is

Reply:
that function is deprecated, it was minimized to allow for a deterministic update, this is safe